pbspro.spec

```diff
@@ -397,22 +397,6 @@ if [ "$1" != "1" ]; then
 	echo
 fi
 
-%posttrans %{pbs_server}
```

Comment:
I agree with @agrawalravi90. Please rebase your branch. This change should not be required.

Reply:
ahh... I'm sorry this change again crept in. I usually push after doing a build and since this is an autogenerated file it becomes part of the commit. I'll rebase in my next push.